databricks/koalas/indexes.py

```diff
@@ -125,6 +125,18 @@ def spark_type(self):
         """ Returns the data type as defined by Spark, as a Spark DataType object."""
         return self.to_series().spark_type
 
+    @property
+    def has_duplicates(self) -> bool:
+        """
+        If index has duplicates, return True, otherwise False.
+        """
+        idx_column = self._kdf._sdf.select(self._scol)
+        deduplicate_idx_column = idx_column.drop_duplicates()
+
+        if idx_column.count() == deduplicate_idx_column.count():
```

Comment:
This runs Spark jobs twice. Can we use `F.count` and `F.countDistinct` instead?

Reply:
ahh! Nice! thanks!!!